python/pyspark/sql/session.py

```diff
@@ -393,6 +393,8 @@ def createDataFrame(self, data, schema=None, samplingRatio=None):
         >>> l = [('Alice', 1)]
         >>> spark.createDataFrame(l).collect()
         [Row(_1=u'Alice', _2=1)]
+        >>> spark.createDataFrame(l, ['name', u'age']).columns
```

Comment:
Oh, it's already tested by python doctest by `runtest.py`.
Do you mean I need to add this into another separate unit test python file?